### PR TITLE
[synapse] Add Feb 2021 release

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

File: releases/2020-02/dotnet.md [hunk 1]
```diff
@@ -89,6 +97,17 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 - Added SMB parameters to File Copy APIs.
 - Added support for file leases.
 
+### Synapse Artifacts [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/synapse/Azure.Analytics.Synapse.Artifacts/CHANGELOG.md)
+- Changed APIs on SparkJobDefinitionClient and SqlScriptClient to provide a Long Running Operation (LRO) when operations can be long in duration.
+- Many models classes are now public.
+- Added BigDataPoolsClient, IntegrationRuntimesClient, SqlPoolsClient, WorkspaceClient and associated support types.
+- Supports List/Get Synapse resources through data plane APIs.
```
Comment (Mohit-Chakraborty) [inline releases/2020-02/dotnet.md:104]: Maybe say 'Support' here as well to be consistent.

**Review by Mohit-Chakraborty — APPROVED** [review]

**Comment by chamons** [comment]
I edited 2020, not 2021 - 🤦

**Review by Mohit-Chakraborty — APPROVED** [review]

File: releases/2021-02/dotnet.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The Azure SDK team is pleased to announce our February 2021 client library relea
 
 #### Updates
 
-- _Add packages_
++- Synapse
```
Comment (jsquire) [inline releases/2021-02/dotnet.md:17]:
```suggestion
- Synapse
```

The current form doesn't appear to be rendering correctly.

File: releases/2021-02/dotnet.md [hunk 3]
```diff
@@ -63,6 +69,16 @@ If you have a bug or feature request for one of the libraries, please [file an i
 
 - Upgraded the `Microsoft.Azure.Amqp` library to resolve crashes occurring in .NET 5.
 
+### Synapse Artifacts [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/synapse/Azure.Analytics.Synapse.Artifacts/CHANGELOG.md)
```
Comment (jsquire) [inline releases/2021-02/dotnet.md:72]: You may want to consider using the common headers described in the [guidelines](https://azure.github.io/azure-sdk/policies_releasenotes.html) for these.

**Review by jsquire — APPROVED** [review]

**Pull request merged**